src/main/java/org/example/bookapp/model/Book.java

```diff
@@ -27,5 +31,8 @@ public class Book {
     @Column(nullable = false)
     private BigDecimal price;
     private String description;
+    @Column(name = "cover-image")
```

Comment:
The naming isn't according to naming convention
Also, don't need to specify the name, Hibernate does it with snake_case